templates/kubeflow/install.html

```diff
@@ -89,40 +83,34 @@ <h3 class="p-stepped-list__title">Enable Kubeflow</h3>
               <input class="p-code-copyable__input" value="microk8s.enable kubeflow" readonly="readonly">
               <button class="p-code-copyable__action">Copy to clipboard</button>
             </div>
+            <p>This will take a few minutes.</p>
           </div>
         </li>
 
         <li class="p-stepped-list__item">
-          <h3 class="p-stepped-list__title">Access Kubeflow</h3>
+          <h3 class="p-stepped-list__title">Access Kubeflow dashboard</h3>
           <div class="p-stepped-list__content">
             <p>
-              If you installed MicroK8s on your local host, then you can use localhost as the IP address in your browser.
-            </p>
-            <p>
-              Otherwise, if you used Multipass as per the instructions above, you can get the IP address of the VM with either <code>multipass list</code> or <code>multipass info kubeflow</code>.
+              After installing kubeflow, an IP address and credentials will be prompted on your terminal. 
             </p>
             <p>
-              Now you should go to your browser and point browser to either:
+              If you installed Microk8s on your local host, you simply need to access the IP address provided on your browser: <code>https://[kubeflow dashboard IP]</kubeflow></code>.
```

Comment:
Closing `</kubeflow>`

Reply:
@anthonydillon can you tell that I'm using a new editor?